src/odemis/odemisd/test/modelget_test.py

```diff
@@ -0,0 +1,148 @@
+import unittest
+import yaml
+
+from odemis.odemisd.modelgen import SafeLoader
+
+
+class SafeLoader_extensions_test(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        SafeLoader.add_constructor('!include', SafeLoader.include)
+        # Overwrite construct_yaml_map with custom mapper
+        SafeLoader.yaml_constructors['tag:yaml.org,2002:map'] = SafeLoader.construct_yaml_map
+        cls.SafeLoader = SafeLoader
```

Comment:
It should be possible to drop all this code once SafeLoader is updated at import.